cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Node.kt

```diff
@@ -208,6 +206,24 @@ open class Node : IVisitable<Node>, Persistable, LanguageProvider, ScopeProvider
     @PopulatedByPass(DFGPass::class, ControlFlowSensitiveDFGPass::class)
     var nextDFG: MutableSet<Node> by PropertyEdgeSetDelegate(Node::nextDFGEdges, true)
 
+    /** Outgoing Program Dependence Edges. */
+    @PopulatedByPass(ProgramDependenceGraphPass::class)
+    @Relationship(value = "PDG", direction = Relationship.Direction.OUTGOING)
+    var nextPDGEdges: MutableList<PropertyEdge<Node>> = mutableListOf()
```

Comment:
I think we should make it a `Set` in accordance with the DFG edges

Reply:
I'm not sure what the conclusion for the DFG edges was, should I use `Sets` for both `nextPDGEdges` and `nextPDG` or only for `nextPDG`?

Reply:
The conclusion was that in some cases (e.g. DFG, EOG, CDG, PDG) `Set`s make more sense because there's no ordering and multiple identical edges don't make sense. We will migrate those in the (near) future. In other cases we will keep `List`s. So I'd suggest to use a `Set` here right away.

Reply:
The conclusion was that in some cases (e.g. DFG, EOG, CDG, PDG) `Set`s make more sense because there's no ordering and multiple identical edges don't make sense. We will migrate those in the (near) future. In other cases we will keep `List`s. So I'd suggest to use a `Set` here right away.